src/main/webapp/app/exam/participate/exercises/quiz/quiz-exam-submission.component.ts

```diff
@@ -67,22 +69,24 @@ export class QuizExamSubmissionComponent extends ExamSubmissionComponent impleme
     }
 
     getExercise(): Exercise {
-        return this.exercise;
+        const quizExercise = new QuizExercise(undefined, undefined);
+        quizExercise.id = this.quizConfiguration.id;
+        return quizExercise;
```

Comment:
Are you sure this works? If the receiver only needs the id and nothing else from the exercise, we should consider changing the getter to `getExerciseId`, otherwise new consumers might think that certain values of the exercise are available while they are not

Reply:
The receiver only needs the exercise id and exercise type. So, yes this works.

Reply:
Yea this probably makes sense

Reply:
Changed the getExercise() method to getExerciseId() and getExerciseType().